app/src/main/java/com/sdpteam/connectout/profile/ProfileEntry.java

```diff
@@ -0,0 +1,22 @@
+package com.sdpteam.connectout.profile;
+
+import java.util.List;
+
+public class ProfileEntry {
+
+    private final Profile profile;
+    private final List<RegisteredEvent> registeredEvents;
+
+    public ProfileEntry(Profile profile, List<RegisteredEvent> registeredEvents) {
+        this.profile = profile;
+        this.registeredEvents = registeredEvents;
+    }
+
+    public Profile getProfile() {
+        return profile;
+    }
+
+    public List<RegisteredEvent> getRegisteredEvents() {
+        return registeredEvents;
+    }
+}
```

Comment:
Please add documentation to this class.
It s impossible to read the code and directly understand what is the use of this code. In fact you said in your PR message that it was a workaround to having multiple query filters in firebase. Maybe this would be a nice place to specify this.